models/models.js

```diff
@@ -37,3 +37,16 @@ exports.getComments = (article_id) => {
     });
 };
 
+//7
+exports.newComment = (comment,article_id) => {
```

Comment:
`newComment` is not a clear enough name for this function - it suggests a variable containing date for a new comment rather than the action of adding or posting a new comment. 
(I'd suggest you rename your controller `postComment` and your model `addComment`